samples/debug/memfault/README.rst

```diff
@@ -15,6 +15,10 @@ To get started with Memfault integration in |NCS|, see :ref:`ug_memfault`.
 Requirements
 ************
 
+Before this sample can be used make sure to have registered a user at `Memfault registration page`_ and `create a new project in Memfault`_.
+You will be directed to the Integration guide for Memfault.
+You will receive the ``project key`` here to set in the :kconfig:option:`CONFIG_MEMFAULT_NCS_PROJECT_KEY` Kconfig option.
```

Comment:
```suggestion
Before using this sample, make sure to complete the steps of Memfault integration into |NCS|. 
See :ref:`ug_memfault` for details.
```
These steps are mentioned in the Memfault integration guide : https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/external_comp/memfault.html#prerequisites . So, we could link the user guide here. We could remove line no:13 as we link the same user guide there also.

Also, we could move the suggested text after the sample yaml table (Step no:24), considering the user flow.

Reply:
Yes, I could agree to that, but at the same time the memfault integration guide takes you through a few more steps with the configuration files, which is already integrated in the sample, so I feel like that at the same time would be a bit misleading. It is really just the prerequisite in the memfault integration guide you need to have set in order for the sample to work.